### PR TITLE
Tag Bio.jl v0.4.6

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/Bio.jl](https://github.com/BioJulia/Bio.jl)
Release: [v0.4.6](https://github.com/BioJulia/Bio.jl/releases/tag/v0.4.6)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/Bio.jl/builds/204206390)
Diff: [vs v0.4.5](https://github.com/BioJulia/Bio.jl/compare/7f5e7afd8ad7c2fab70b812f6596e9163b9eca70...6cf068fc7f888a67dc8d1eb1efa44bfb7c1ec96e)
`requires` vs v0.4.5: 
```diff
--- 0.4.5/requires
+++ 0.4.6/requires
@@ -4,7 +4,7 @@
 Colors
 Distributions
 IndexableBitVectors 0.1.1
-IntervalTrees 0.1
+IntervalTrees 0.1 0.1.1
 Iterators
 Libz 0.2
 LightGraphs
```
cc: @Ward9250

### CODE REVIEW

File: Bio/versions/0.4.6/requires [hunk 1]
```diff
@@ -0,0 +1,13 @@
+julia 0.5
+BGZFStreams 0.1
+BufferedStreams 0.2
+Colors
+Distributions
+IndexableBitVectors 0.1.1
+IntervalTrees 0.1 0.1.1
```
Comment (tkelman) [inline Bio/versions/0.4.6/requires]: It actually doesn't help much to just tag a new version with the upper bound, Pkg can just downgrade Bio to an older version without the bound. You'd have to make a PR by hand that adds the bound to the `requires` file (only the METADATA copy) for all the past versions of Bio.jl

Comment (Ward9250) [inline Bio/versions/0.4.6/requires]: Ok, I do that separately in another PR? This release is made now, and it takes another release to undo this one.

Comment (tkelman) [inline Bio/versions/0.4.6/requires]: yes, separate pr

it hasn't been merged or published in metadata, so you can still edit the release without doing much harm

Comment (Ward9250) [inline Bio/versions/0.4.6/requires]: Ok I've cancelled this release, and will make a separate PR manually editing requires for each version. I trust even though the REQUIRE file in the package may not always match, it will not be a problem.

Comment (tkelman) [inline Bio/versions/0.4.6/requires]: the metadata version of the require file takes precedence for tagged versions of registered packages, they can diverge - don't touch the sha1 contents though

Comment (Ward9250) [inline Bio/versions/0.4.6/requires]: Ok, PR made, only the require file have been edited - no edits of the sha files.

Comment (tkelman) [inline Bio/versions/0.4.6/requires]: so should this PR #8064 be closed then?

Comment (Ward9250) [inline Bio/versions/0.4.6/requires]: Yes please, the latest PR to edit the requires should clear everything up.
